src/Microsoft.ML.ImageAnalytics/ImagePixelExtractor.cs

```diff
@@ -530,24 +583,22 @@ public sealed class ColumnInfo
             /// <summary> What colors to extract.</summary>
             public readonly ColorBits Colors;
 
-            /// <summary>Offset color pixel value by this amount.</summary>
+            /// <summary> In which color order extract values from pixel.</summary>
+            public readonly ColorsOrder Order;
+
+            /// <summary>Offset pixel's color value by this amount. Applied to color value first.</summary>
             public readonly float Offset;
 
-            /// <summary>Scale color pixel value by this amount.</summary>
+            /// <summary>Scale pixel's color value by this amount. Applied to color value second.</summary>
             public readonly float Scale;
 
-            /// <summary> Whether to interleave the pixels, meaning keep them in the `ARGB ARGB` order, or leave them separated in the plannar form.</summary>
+            /// <summary>Whether to interleave the pixels colors, meaning keep them in the <see cref="Order"/> order, or leave them in the plannar form.</summary>
```

Comment:
>Whether to interleave the pixels colors, meaning keep them in the <see cref="Order"/> order, or leave them in the plannar form. [](start = 25, length = 127)

Could you double check with the catalog to make sure we have something consistent for this line?